js.js

```diff
@@ -0,0 +1,57 @@
+const urlServer = "http://localhost:5050/fibonacci/" ;
+document.getElementById("btn").addEventListener("click", fibonacciServer );
+let errorMesage = document.getElementById("error");
+
+//document.getElementById("btn").addEventListener("click", spinner);
+
+function spinnerOn() {
+    var element = document.getElementById("spiner");
+    element.classList.add("d-block")
+    element.classList.remove("d-none");
+ }
+ function spinnerOff() {
+    var element = document.getElementById("spiner");
+    element.classList.add("d-none");
+ }
+ spinnerOff()
+
+function fibonacciServer(){
```

Comment:
Some logic (if(userNum>50), if (userNum == 42) && if (userNum < 0) is written twice in 2 different funcitons. Instead, you should write is only in 1 place and then call the right function. Remember DRY principle.